package.json

```diff
@@ -47,10 +57,12 @@
     "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-tailwindcss": "^3.13.0",
+    "nodemon": "^3.0.3",
```

Comment:
Will we use nodemon for custom server? I don't see any nodemon command in the running script.

Reply:
we actually don't need this, i'm going to remove it